aspnetcore/blazor/components/quickgrid.md

```diff
@@ -138,6 +138,63 @@ To provide a UI for pagination, add a [`Paginator` component](xref:Microsoft.Asp
 
 In the running app, page through the items using a rendered `Paginator` component.
 
+QuickGrid renders additional empty rows to fill in the final page of data when used with a `Paginator` component. In .NET 9 or later, empty data cells (`<td></td>`) are added to the empty rows. The empty rows are intended to facilitate rendering the QuickGrid with stable row height and styling across all pages. You can apply styles to the rows using [CSS isolation](xref:blazor/components/css-isolation) by wrapping the `QuickGrid` component in a wrapper element, such as a `<div>`, and applying a row style with `::deep` [pseudo-elements](https://developer.mozilla.org/docs/Web/CSS/Pseudo-elements):
+
+```css
+::deep tr {
+    height: 2em;
+}
+```
+
+:::moniker range=">= aspnetcore-9.0"
+
+<!-- UPDATE 10.0 Check on https://github.com/dotnet/aspnetcore/issues/59078 to see
```

Comment:
I believe this feature request is actually about having a template to render when no records are bound, so the QuickGrid is empty of any data.

Reply:
Check out his opening paragraph ... I think the empty record row template styling would allow styling to prevent these rendered rows from showing the borders.

https://github.com/dotnet/aspnetcore/issues/59078